gnd/src/main/java/com/google/android/gnd/ui/home/featuresheet/RecordListItemViewHolder.java

```diff
@@ -18,74 +18,46 @@
 
 import android.content.Context;
 import android.content.res.Resources;
-import android.text.TextUtils.TruncateAt;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.TableRow;
+import android.text.TextUtils;
 import android.widget.TextView;
 import androidx.annotation.NonNull;
-import androidx.lifecycle.MutableLiveData;
 import androidx.recyclerview.widget.RecyclerView;
-import butterknife.BindView;
-import butterknife.ButterKnife;
 import com.google.android.gnd.R;
+import com.google.android.gnd.databinding.RecordListItemBinding;
 import com.google.android.gnd.model.form.Element;
 import com.google.android.gnd.model.form.Field;
 import com.google.android.gnd.model.form.Form;
 import com.google.android.gnd.model.observation.Record;
 import com.google.android.gnd.model.observation.Response;
-import com.google.android.gnd.system.AuthenticationManager;
-import java.text.DateFormat;
-import java.util.Date;
+import com.google.android.gnd.ui.common.Consumer;
 import java8.util.Optional;
 
 class RecordListItemViewHolder extends RecyclerView.ViewHolder {
-  private static final int MAX_SUMMARY_COLUMNS = 4;
-  private final View view;
-  private final MutableLiveData<Record> itemClicks;
-
-  @BindView(R.id.user_name)
-  TextView userNameView;
-
-  @BindView(R.id.last_modified_date)
-  TextView lastModifiedDateView;
 
-  @BindView(R.id.last_modified_time)
-  TextView lastModifiedTimeView;
-
-  @BindView(R.id.field_label_row)
-  TableRow fieldLabelRow;
+  private static final int MAX_SUMMARY_COLUMNS = 4;
 
-  @BindView(R.id.field_value_row)
-  TableRow fieldValueRow;
+  private final Consumer<Record> consumer;
```

Comment:
`consumer` is the higher-order type; I'd name this more specifically `recordCallback` or similar.

Reply:
Done